Test.System.Containers.Filesystem/EndToEnd.cs

```diff
@@ -194,10 +193,6 @@ public async Task EndToEnd_NoAPI()
         await publish.WaitForExitAsync();
         Assert.AreEqual(0, publish.ExitCode);
 
-        // Final step: verify by running the app.
-        // This is not as simple as checking the output contains "hello world" because
-        // we can't publish a console app (the container targets only import in websdk projects)
```

Comment:
Is this not true any more?

Reply:
This is still true, think we should keep the comment in for context?

Reply:
You can publish non-webapps, and if you explicitly specify the DefaultContainer profile then all of our stuff gets imported as expected. So this comment may not be entirely correct.

Reply:
The demo app in the INTEGRATED-DEMO file is a console app, for example.